package.json

```diff
@@ -11,8 +11,8 @@
     "@uifabric/fluent-theme": "^0.16.7",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.0",
-    "typescript": "3.4.5"
+    "react-scripts": "3.4.4",
+    "typescript": "3.8.3"
```

Comment:
Can we go ahead and update this to 3.9 or later? Ideally 4.1 if it works okay.

Reply:
AFAIR typescript v4 is supported starting CRA v4, I'd suggest to bump/migrate to that versions after initial fixes are done. WDYT?

Reply:
3.9 is fine in that case